### ISSUE TITLE
[UT]tck error

### ISSUE THREAD

**Issue by Shylock-Hg** [vesoft-inc/nebula-graph#946]
https://github.com/vesoft-inc/nebula-graph/runs/2331539126?check_suite_focus=true

```
PYTHONPATH=$PYTHONPATH:/__w/nebula-graph/nebula-graph/tests/.. /__w/nebula-graph/nebula-graph/tests/nebula-test-run.py --cmd=start --rm_dir=true --build_dir=/__w/nebula-graph/nebula-graph/tests/../build
[2021-04-13 16:24:45,568]:Get connection to ('127.0.0.1', 6459)
work directory: /__w/nebula-graph/nebula-graph/tests/../build/server_2021-04-13T16-24-41
exec: bin/nebula-metad --flagfile conf/nebula-metad.conf --meta_server_addrs=127.0.0.1:7460 --port=7460 --ws_http_port=5000 --ws_h2_port=1469 --heartbeat_interval_secs=1 --expired_time_factor=60 --v=4
exec: bin/nebula-storaged --flagfile conf/nebula-storaged.conf --meta_server_addrs=127.0.0.1:7460 --port=8306 --ws_http_port=5833 --ws_h2_port=9896 --heartbeat_interval_secs=1 --expired_time_factor=60 --raft_heartbeat_interval_secs=30 --v=4
exec: bin/nebula-graphd --flagfile conf/nebula-graphd.conf --meta_server_addrs=127.0.0.1:7460 --port=6459 --ws_http_port=9106 --ws_h2_port=7589 --heartbeat_interval_secs=1 --expired_time_factor=60 --enable_optimizer=true --enable_authorize=true --v=4
Start nebula successfully
cd /__w/nebula-graph/nebula-graph/tests && python3 -m pytest -n6 --dist=loadfile --rm_dir=false -m "not skip" -k "not tck" /__w/nebula-graph/nebula-graph/tests
============================= test session starts ==============================
platform linux -- Python 3.6.9, pytest-5.3.2, py-1.9.0, pluggy-0.13.1 -- /usr/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.6.9', 'Platform': 'Linux-5.4.109-1.el7.elrepo.x86_64-x86_64-with-Ubuntu-18.04-bionic', 'Packages': {'pytest': '5.3.2', 'py': '1.9.0', 'pluggy': '0.13.1'}, 'Plugins': {'metadata': '1.8.0', 'html': '2.0.1', 'reportlog': '0.1.0', 'forked': '1.3.0', 'xdist': '1.31.0', 'benchmark': '3.2.3', 'drop-dup-tests': '0.3.0', 'bdd': '4.0.1', 'yapf3': '0.5.1'}, 'CI': 'true'}
benchmark: 3.2.3 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /__w/nebula-graph/nebula-graph/tests, inifile: pytest.ini
plugins: metadata-1.8.0, html-2.0.1, reportlog-0.1.0, forked-1.3.0, xdist-1.31.0, benchmark-3.2.3, drop-dup-tests-0.3.0, bdd-4.0.1, yapf3-0.5.1
gw0 I / gw1 I / gw2 I / gw3 I / gw4 I / gw5 I

[gw0] linux Python 3.6.9 cwd: /__w/nebula-graph/nebula-graph/tests

[gw1] linux Python 3.6.9 cwd: /__w/nebula-graph/nebula-graph/tests

[gw2] linux Python 3.6.9 cwd: /__w/nebula-graph/nebula-graph/tests

[gw3] linux Python 3.6.9 cwd: /__w/nebula-graph/nebula-graph/tests

[gw4] linux Python 3.6.9 cwd: /__w/nebula-graph/nebula-graph/tests

[gw5] linux Python 3.6.9 cwd: /__w/nebula-graph/nebula-graph/tests

[gw0] Python 3.6.9 (default, Jan 26 2021, 15:33:00)  -- [GCC 8.4.0]

[gw1] Python 3.6.9 (default, Jan 26 2021, 15:33:00)  -- [GCC 8.4.0]

[gw2] Python 3.6.9 (default, Jan 26 2021, 15:33:00)  -- [GCC 8.4.0]
Generating LALR tables
Generating LALR tables

[gw3] Python 3.6.9 (default, Jan 26 2021, 15:33:00)  -- [GCC 8.4.0]

[gw4] Python 3.6.9 (default, Jan 26 2021, 15:33:00)  -- [GCC 8.4.0]

[gw5] Python 3.6.9 (default, Jan 26 2021, 15:33:00)  -- [GCC 8.4.0]
gw0 [47] / gw1 [47] / gw2 [0] / gw3 [47] / gw4 [47] / gw5 [47]

scheduling tests via LoadFileScheduling

==================================== ERRORS ====================================
________________________ ERROR collecting test session _________________________
/github/home/.local/lib/python3.6/site-packages/_pytest/config/__init__.py:458: in _importconftest
    return self._conftestpath2mod[key]
E   KeyError: PosixPath('/__w/nebula-graph/nebula-graph/tests/tck/conftest.py')

During handling of the above exception, another exception occurred:
/github/home/.local/lib/python3.6/site-packages/_pytest/config/__init__.py:464: in _importconftest
    mod = conftestpath.pyimport()
/github/home/.local/lib/python3.6/site-packages/py/_path/local.py:704: in pyimport
    __import__(modname)
<frozen importlib._bootstrap>:971: in _find_and_load
    ???
<frozen importlib._bootstrap>:955: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:665: in _load_unlocked
    ???
/github/home/.local/lib/python3.6/site-packages/_pytest/assertion/rewrite.py:143: in exec_module
    exec(co, module.__dict__)
tck/conftest.py:31: in <module>
    from tests.tck.utils.table import dataset, table
tck/utils/table.py:10: in <module>
    from tests.tck.utils.nbv import parse
tck/utils/nbv.py:546: in <module>
    parser = yacc.yacc()
/github/home/.local/lib/python3.6/site-packages/ply/yacc.py:3293: in yacc
    read_signature = lr.read_table(tabmodule)
/github/home/.local/lib/python3.6/site-packages/ply/yacc.py:1987: in read_table
    if parsetab._tabversion != __tabversion__:
E   AttributeError: module 'tests.tck.utils.parsetab' has no attribute '_tabversion'

During handling of the above exception, another exception occurred:
/github/home/.local/lib/python3.6/site-packages/py/_path/common.py:383: in visit
    for x in Visitor(fil, rec, ignore, bf, sort).gen(self):
/github/home/.local/lib/python3.6/site-packages/py/_path/common.py:424: in gen
    dirs = self.optsort([p for p in entries
/github/home/.local/lib/python3.6/site-packages/py/_path/common.py:425: in <listcomp>
    if p.check(dir=1) and (rec is None or rec(p))])
/github/home/.local/lib/python3.6/site-packages/_pytest/main.py:622: in _recurse
    ihook = self.gethookproxy(dirpath)
/github/home/.local/lib/python3.6/site-packages/_pytest/main.py:441: in gethookproxy
    my_conftestmodules = pm._getconftestmodules(fspath)
/github/home/.local/lib/python3.6/site-packages/_pytest/config/__init__.py:436: in _getconftestmodules
    mod = self._importconftest(conftestpath)
/github/home/.local/lib/python3.6/site-packages/_pytest/config/__init__.py:472: in _importconftest
    raise ConftestImportFailure(conftestpath, sys.exc_info())
E   _pytest.config.ConftestImportFailure: (local('/__w/nebula-graph/nebula-graph/tests/tck/conftest.py'), (<class 'AttributeError'>, AttributeError("module 'tests.tck.utils.parsetab' has no attribute '_tabversion'",), <traceback object at 0x7f20d01e8608>))
_____________________________ ERROR collecting gw1 _____________________________
Different tests were collected between gw2 and gw1. The difference is:
--- gw2

+++ gw1

@@ -0,0 +1,47 @@

+admin/test_charset.py::TestCharset::test_show_charset
+admin/test_charset.py::TestCharset::test_show_collate
+admin/test_configs.py::TestConfigs::test_configs
+admin/test_listener.py::TestListener::test_listener
+admin/test_parts.py::TestParts::test_part
+admin/test_permission.py::TestPermission::test_simple
+admin/test_permission.py::TestPermission::test_user_write
+admin/test_permission.py::TestPermission::test_schema_and_data
+admin/test_permission.py::TestPermission::test_show_test
+admin/test_permission.py::TestPermission::test_show_roles
+admin/test_show_hosts.py::TestShowHosts::test_show_hosts
+admin/test_space.py::TestSpace::test_space
+admin/test_space.py::TestSpace::test_charset_collate
+admin/test_space.py::TestSpace::test_if_not_exists_and_if_exist
+admin/test_space.py::TestSpace::test_drop_space
+admin/test_space.py::TestSpace::test_create_space_with_string_vid
+admin/test_space.py::TestSpace::test_create_space_with_int_vid
+admin/test_users.py::TestUsers::test_create_users
+admin/test_users.py::TestUsers::test_alter_users
+admin/test_users.py::TestUsers::test_drop_users
+admin/test_users.py::TestUsers::test_change_password
+admin/test_users.py::TestUsers::test_grant_revoke
+maintain/test_comments.py::TestComment::test_comment
+maintain/test_zone.py::TestZone::test_zone
+query/stateless/test_admin.py::TestAdmin::test_config
+query/stateless/test_go.py::TestGoQuery::test_1_step
+query/stateless/test_go.py::TestGoQuery::test_2_steps
+query/stateless/test_if_exists.py::TestDropSpaceIfExists::test_drop_space
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords1
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords2
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords3
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[1]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[2048]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[4096]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4097]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4444]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[10240]
+query/stateless/test_schema.py::TestSchema::test_create_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_create_tag_failed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_failed
+query/stateless/test_schema.py::TestSchema::test_create_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_create_edge_failed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_failed
+query/stateless/test_schema.py::TestSchema::test_cover_fix_negative_default_value
+query/stateless/test_schema.py::TestSchema::test_default_expression_value
_____________________________ ERROR collecting gw0 _____________________________
Different tests were collected between gw2 and gw0. The difference is:
--- gw2

+++ gw0

@@ -0,0 +1,47 @@

+admin/test_charset.py::TestCharset::test_show_charset
+admin/test_charset.py::TestCharset::test_show_collate
+admin/test_configs.py::TestConfigs::test_configs
+admin/test_listener.py::TestListener::test_listener
+admin/test_parts.py::TestParts::test_part
+admin/test_permission.py::TestPermission::test_simple
+admin/test_permission.py::TestPermission::test_user_write
+admin/test_permission.py::TestPermission::test_schema_and_data
+admin/test_permission.py::TestPermission::test_show_test
+admin/test_permission.py::TestPermission::test_show_roles
+admin/test_show_hosts.py::TestShowHosts::test_show_hosts
+admin/test_space.py::TestSpace::test_space
+admin/test_space.py::TestSpace::test_charset_collate
+admin/test_space.py::TestSpace::test_if_not_exists_and_if_exist
+admin/test_space.py::TestSpace::test_drop_space
+admin/test_space.py::TestSpace::test_create_space_with_string_vid
+admin/test_space.py::TestSpace::test_create_space_with_int_vid
+admin/test_users.py::TestUsers::test_create_users
+admin/test_users.py::TestUsers::test_alter_users
+admin/test_users.py::TestUsers::test_drop_users
+admin/test_users.py::TestUsers::test_change_password
+admin/test_users.py::TestUsers::test_grant_revoke
+maintain/test_comments.py::TestComment::test_comment
+maintain/test_zone.py::TestZone::test_zone
+query/stateless/test_admin.py::TestAdmin::test_config
+query/stateless/test_go.py::TestGoQuery::test_1_step
+query/stateless/test_go.py::TestGoQuery::test_2_steps
+query/stateless/test_if_exists.py::TestDropSpaceIfExists::test_drop_space
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords1
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords2
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords3
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[1]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[2048]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[4096]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4097]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4444]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[10240]
+query/stateless/test_schema.py::TestSchema::test_create_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_create_tag_failed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_failed
+query/stateless/test_schema.py::TestSchema::test_create_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_create_edge_failed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_failed
+query/stateless/test_schema.py::TestSchema::test_cover_fix_negative_default_value
+query/stateless/test_schema.py::TestSchema::test_default_expression_value
_____________________________ ERROR collecting gw4 _____________________________
Different tests were collected between gw2 and gw4. The difference is:
--- gw2

+++ gw4

@@ -0,0 +1,47 @@

+admin/test_charset.py::TestCharset::test_show_charset
+admin/test_charset.py::TestCharset::test_show_collate
+admin/test_configs.py::TestConfigs::test_configs
+admin/test_listener.py::TestListener::test_listener
+admin/test_parts.py::TestParts::test_part
+admin/test_permission.py::TestPermission::test_simple
+admin/test_permission.py::TestPermission::test_user_write
+admin/test_permission.py::TestPermission::test_schema_and_data
+admin/test_permission.py::TestPermission::test_show_test
+admin/test_permission.py::TestPermission::test_show_roles
+admin/test_show_hosts.py::TestShowHosts::test_show_hosts
+admin/test_space.py::TestSpace::test_space
+admin/test_space.py::TestSpace::test_charset_collate
+admin/test_space.py::TestSpace::test_if_not_exists_and_if_exist
+admin/test_space.py::TestSpace::test_drop_space
+admin/test_space.py::TestSpace::test_create_space_with_string_vid
+admin/test_space.py::TestSpace::test_create_space_with_int_vid
+admin/test_users.py::TestUsers::test_create_users
+admin/test_users.py::TestUsers::test_alter_users
+admin/test_users.py::TestUsers::test_drop_users
+admin/test_users.py::TestUsers::test_change_password
+admin/test_users.py::TestUsers::test_grant_revoke
+maintain/test_comments.py::TestComment::test_comment
+maintain/test_zone.py::TestZone::test_zone
+query/stateless/test_admin.py::TestAdmin::test_config
+query/stateless/test_go.py::TestGoQuery::test_1_step
+query/stateless/test_go.py::TestGoQuery::test_2_steps
+query/stateless/test_if_exists.py::TestDropSpaceIfExists::test_drop_space
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords1
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords2
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords3
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[1]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[2048]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[4096]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4097]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4444]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[10240]
+query/stateless/test_schema.py::TestSchema::test_create_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_create_tag_failed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_failed
+query/stateless/test_schema.py::TestSchema::test_create_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_create_edge_failed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_failed
+query/stateless/test_schema.py::TestSchema::test_cover_fix_negative_default_value
+query/stateless/test_schema.py::TestSchema::test_default_expression_value
_____________________________ ERROR collecting gw3 _____________________________
Different tests were collected between gw2 and gw3. The difference is:
--- gw2

+++ gw3

@@ -0,0 +1,47 @@

+admin/test_charset.py::TestCharset::test_show_charset
+admin/test_charset.py::TestCharset::test_show_collate
+admin/test_configs.py::TestConfigs::test_configs
+admin/test_listener.py::TestListener::test_listener
+admin/test_parts.py::TestParts::test_part
+admin/test_permission.py::TestPermission::test_simple
+admin/test_permission.py::TestPermission::test_user_write
+admin/test_permission.py::TestPermission::test_schema_and_data
+admin/test_permission.py::TestPermission::test_show_test
+admin/test_permission.py::TestPermission::test_show_roles
+admin/test_show_hosts.py::TestShowHosts::test_show_hosts
+admin/test_space.py::TestSpace::test_space
+admin/test_space.py::TestSpace::test_charset_collate
+admin/test_space.py::TestSpace::test_if_not_exists_and_if_exist
+admin/test_space.py::TestSpace::test_drop_space
+admin/test_space.py::TestSpace::test_create_space_with_string_vid
+admin/test_space.py::TestSpace::test_create_space_with_int_vid
+admin/test_users.py::TestUsers::test_create_users
+admin/test_users.py::TestUsers::test_alter_users
+admin/test_users.py::TestUsers::test_drop_users
+admin/test_users.py::TestUsers::test_change_password
+admin/test_users.py::TestUsers::test_grant_revoke
+maintain/test_comments.py::TestComment::test_comment
+maintain/test_zone.py::TestZone::test_zone
+query/stateless/test_admin.py::TestAdmin::test_config
+query/stateless/test_go.py::TestGoQuery::test_1_step
+query/stateless/test_go.py::TestGoQuery::test_2_steps
+query/stateless/test_if_exists.py::TestDropSpaceIfExists::test_drop_space
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords1
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords2
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords3
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[1]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[2048]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[4096]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4097]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4444]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[10240]
+query/stateless/test_schema.py::TestSchema::test_create_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_create_tag_failed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_failed
+query/stateless/test_schema.py::TestSchema::test_create_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_create_edge_failed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_failed
+query/stateless/test_schema.py::TestSchema::test_cover_fix_negative_default_value
+query/stateless/test_schema.py::TestSchema::test_default_expression_value
_____________________________ ERROR collecting gw5 _____________________________
Different tests were collected between gw2 and gw5. The difference is:
--- gw2

+++ gw5

@@ -0,0 +1,47 @@

+admin/test_charset.py::TestCharset::test_show_charset
+admin/test_charset.py::TestCharset::test_show_collate
+admin/test_configs.py::TestConfigs::test_configs
+admin/test_listener.py::TestListener::test_listener
+admin/test_parts.py::TestParts::test_part
+admin/test_permission.py::TestPermission::test_simple
+admin/test_permission.py::TestPermission::test_user_write
+admin/test_permission.py::TestPermission::test_schema_and_data
+admin/test_permission.py::TestPermission::test_show_test
+admin/test_permission.py::TestPermission::test_show_roles
+admin/test_show_hosts.py::TestShowHosts::test_show_hosts
+admin/test_space.py::TestSpace::test_space
+admin/test_space.py::TestSpace::test_charset_collate
+admin/test_space.py::TestSpace::test_if_not_exists_and_if_exist
+admin/test_space.py::TestSpace::test_drop_space
+admin/test_space.py::TestSpace::test_create_space_with_string_vid
+admin/test_space.py::TestSpace::test_create_space_with_int_vid
+admin/test_users.py::TestUsers::test_create_users
+admin/test_users.py::TestUsers::test_alter_users
+admin/test_users.py::TestUsers::test_drop_users
+admin/test_users.py::TestUsers::test_change_password
+admin/test_users.py::TestUsers::test_grant_revoke
+maintain/test_comments.py::TestComment::test_comment
+maintain/test_zone.py::TestZone::test_zone
+query/stateless/test_admin.py::TestAdmin::test_config
+query/stateless/test_go.py::TestGoQuery::test_1_step
+query/stateless/test_go.py::TestGoQuery::test_2_steps
+query/stateless/test_if_exists.py::TestDropSpaceIfExists::test_drop_space
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords1
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords2
+query/stateless/test_keyword.py::TestReservedKeyword::test_keywords3
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[1]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[2048]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_valid[4096]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4097]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[4444]
+query/stateless/test_range.py::TestRangeChecking::test_label_length_invalid[10240]
+query/stateless/test_schema.py::TestSchema::test_create_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_create_tag_failed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_tag_failed
+query/stateless/test_schema.py::TestSchema::test_create_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_create_edge_failed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_succeed
+query/stateless/test_schema.py::TestSchema::test_alter_edge_failed
+query/stateless/test_schema.py::TestSchema::test_cover_fix_negative_default_value
+query/stateless/test_schema.py::TestSchema::test_default_expression_value
=============================== warnings summary ===============================
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:760
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:760
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:760
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:760
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:760
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:760
  /github/home/.local/lib/python3.6/site-packages/ply/lex.py:760: DeprecationWarning: Flags not at the start of the expression '(?P<t_BOOLEAN>(?i)tr' (truncated)
    c = re.compile('(?P<%s>%s)' % (fname, _get_regex(f)), self.reflags)

/github/home/.local/lib/python3.6/site-packages/ply/lex.py:498
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:498
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:498
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:498
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:498
/github/home/.local/lib/python3.6/site-packages/ply/lex.py:498
  /github/home/.local/lib/python3.6/site-packages/ply/lex.py:498: DeprecationWarning: Flags not at the start of the expression '(?P<t_EMPTY>EMPTY)|(' (truncated)
    lexre = re.compile(regex, reflags)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
/github/home/.local/lib/python3.6/site-packages/pytest_benchmark/logger.py:44: PytestBenchmarkWarning: Benchmarks are automatically disabled because xdist plugin is active.Benchmarks cannot be performed reliably in a parallelized environment.
  warner(PytestBenchmarkWarning(text))
=========================== short test summary info ============================
ERROR  - _pytest.config.ConftestImportFailure: (local('/__w/nebula-graph/nebu...
ERROR gw1
ERROR gw0
ERROR gw4
ERROR gw3
ERROR gw5
======================== 12 warnings, 6 errors in 2.03s ========================
make: *** [test] Error 1
Makefile:52: recipe for target 'test' failed

```

**Comment by yixinglu** [comment]
duplicates with #941

**Comment by CPWstatic** [comment]
Please give a clear message next time. The link may be broken.

**Comment by yixinglu** [comment]
Try to fixed in #1108. If you encounter this error later, please let me know here.

close it now.
